_IBC/furya-osmosis.json

```diff
@@ -2,22 +2,22 @@
   "$schema": "../ibc_data.schema.json",
   "chain_1": {
     "chain_name": "furya",
-    "client_id": "07-tendermint-4",
-    "connection_id": "connection-3"
+    "client_id": "07-tendermint-10",
+    "connection_id": "connection-9"
   },
   "chain_2": {
     "chain_name": "osmosis",
-    "client_id": "07-tendermint-3029",
-    "connection_id": "connection-2515"
+    "client_id": "07-tendermint-3201",
+    "connection_id": "connection-2658"
   },
   "channels": [
     {
       "chain_1": {
-        "channel_id": "channel-3",
+        "channel_id": "channel-8",
         "port_id": "transfer"
       },
       "chain_2": {
-        "channel_id": "channel-8690",
+        "channel_id": "channel-74222",
```

Comment:
There are already existing IBC connections that have been registered for a long time. Why are new connections being registered? Can we prove that both of these connections have no abandoned tokens on the counterparty chain?

Reply:
Creating liquidity pools on Osmosis requires having a reliable channel open to the Osmosis chain. We find the new relayers to be reliable. The one for Terra2 will also be currently managed by the same provider.
I have asked for a confirmation from the providers about the any abandoned assets and a proper halting of the older relayer. I'll keep you posted.

Reply:
Can these reliable relayers not just operate on the existing channels?

Reply:
Previous channels expired, so new ones had to be created anyway. At the moment it is just under discussion with the team if we should raise governance to replace old with new or just use new channels.

Reply:
I have osmosisd installed, and it looks like there is a total of 1.000009 FURY on Osmosis, which is much less than 1 penny, and so can be considered a negligible 'testing amount'.
I'm not sure how much OSMO is on Furya. If we can show that's it's also a tiny amount then we should be able to update.

Reply:
It has also been determined that no Osmosis assets are on Furya, so I think we are OK to proceed with new channels. Is this still desired?

Reply:
Yes, please. Let's proceed with the new channels.